### PR TITLE
Voice control by Alexa

### PR DESCRIPTION
**Pull request by sctse999**
This part works together with a public Alexa skill that we have released, enabling voice control to start, stop, speed up and slow down an autopilot. For details, please check the README.md.

### CODE REVIEW

**Comment by tawnkramer** [comment]
excited to try this out! thanks

File: donkeycar/parts/voice_control/alexa.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            raise Exception("Please set cfg.ALEXA_DEVICE_CODE in myconfig.py")
+
+    def get_command(self):
+        url = "{}/{}".format(self.API_ENDPOINT, 'command')
+
+        params = {
+            'deviceCode': self.cfg.ALEXA_DEVICE_CODE
+        }
+
+        r = requests.get(url=url, params=params)
```
Comment (nvtkaszpir) [inline donkeycar/parts/voice_control/alexa.py]: I suggest to add some sane timeout and retry system in here.

File: donkeycar/parts/voice_control/alexa.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+            if command == "autopilot":
+                self.ctr.mode = "local"
+            elif command == "speedup":
+                self.cfg.AI_THROTTLE_MULT += 0.05
+            elif command == "slowdown":
+                self.cfg.AI_THROTTLE_MULT -= 0.05
+            elif command == "stop":
+                self.ctr.mode = "user"
+                self.cfg.AI_THROTTLE_MULT = 1
```
Comment (nvtkaszpir) [inline donkeycar/parts/voice_control/alexa.py]: Shouldn't it revert to the default setting that was in the beginning?
Certain cars will not move at all if AI_THROTTLE_MULT is set to 1

File: donkeycar/parts/voice_control/alexa.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+            if self.debug:
+                print("mode = {}, cfg.AI_THROTTLE_MULT={}".format(
+                    self.ctr.mode, self.cfg.AI_THROTTLE_MULT))
+            time.sleep(0.25)
```
Comment (nvtkaszpir) [inline donkeycar/parts/voice_control/alexa.py]: Maybe a comment why `sleep` was added?

File: donkeycar/parts/voice_control/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+# Alexa Support
```
[the remark this answers is not in the document]
Comment (sctse999) [reply donkeycar/parts/voice_control/README.md]: You are right. Let me move it under doc

**Pull request merged**